lib/mongoid/config/options.rb

```diff
@@ -36,8 +36,16 @@ def option(name, options = {})
             end
           end
 
-          define_method("#{name}=") do |value|
-            settings[name] = value
+          if name.to_sym == :broken_view_options
```

Comment:
I would do something like `name = name.to_sym` one time in this method right in the beginning.

Reply:
Also, the docstring says name can only be a symbol, so either this conversion is unnecessary or the docstring should be amended to specify string as allowed input.

Reply:
Probably the former.

Reply:
done